scripts/managedb/csubs_control/README.md

```diff
@@ -0,0 +1,20 @@
+This directory includes some scripts to help to control context subscriptions collection (csubs). All them share
+the same CLI arguments:
+
+* `-db <database>` (mandatory): to specify the DB to process
+* `--dry-run (optional): it avoids actual modification in the DB, just reporting
+* `--with-restart <status_file>` (optional): restart Context Broker service if the database was manipulated and,
+   consecuently, Context Broker needs to be restarted. It is based in `/etc/init.d/contextBroker` service script,
+   so it will not work if the CB was not installed as RPM. It requires root permissions.
+* `-v` (optional): enables verbose mode
+
+The scripts are:
+
+* csub_dups_checker.py: removes csubs duplicates. Have a look to script implementation in order to know which
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/7115621dcc90ba36dee2235ad4ccc909ee1b11d3